src/main/java/com/dongtronic/diabot/commands/NightscoutCommand.kt

```diff
@@ -243,6 +246,30 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand() {
         val jsonObject = JsonParser().parse(json).asJsonArray.get(0).asJsonObject
         val sgv = jsonObject.get("sgv").asString
         val timestamp = jsonObject.get("date").asLong
+        val trend: String = ""
+        val driection: String = ""
+        if (jsonObject.has("trend")) {
+            val trend = jsonObject.get("trend").asInt
+        }
+        else {
+            if (jsonObject.has("direction")) {
+                val direction = jsonObject.get("direction").asString
+                val trend = when (direction) {
```

Comment:
No need to reinitialize the variable again since you did so on line 249 already (please remove val)

Reply:
Should be good.